e2e-tests/development-runtime/cypress/integration/styling/less.js

```diff
@@ -24,12 +24,11 @@ describe(`styling: less`, () => {
   })
 
   describe(`hot reloading`, () => {
-    it(`plain less`, () => {
-      // we don't want to visit page for each test - we want to visit once and then test HMR
-      cy.window().then(win => {
-        cy.spy(win.console, `log`).as(`hmrConsoleLog`)
```

Comment:
This is already getting set inside the `support/e2e.ts` file with the

```
Cypress.Commands.overwrite("visit")
```

overwrite